README.md

```diff
@@ -122,6 +126,33 @@ services:
       - LLDAP_LDAP_BASE_DN=dc=example,dc=com
 ```
 
+Example for docker compose for `:latest` tag:
```

Comment:
Can you add a couple of words on the difference?

Reply:
This is because our `:latest` using UID:GID  when people it also expected it working at stable. Can we create a release like `0.4.1` so the UID:GID consistently implemented in our image. Then we only need to change the current compose using UID:GID?